CONTRIBUTING.md

```diff
@@ -1,53 +1,80 @@
-# Giant Swarm docs contriubtion guidelines
+# Giant Swarm docs contribution guidelines
 
 **We welcome any contributions on content to this repository in the form of pull requests!**
 
 Please review these contribution guidelines to ease getting your changes into this repository.
 
 ## Style
 
-### Using acronyms
+Some general guidelines to know about when editing or reviewing content.
 
-Please avoid acronyms and abbreviations where possible, use them only where the acronym is easier to understand than the long form (example: `SSH` is self-explanatory, `secure shell` is not widely understood).
+### Acronyms and abbreviations
 
-When you want to use an acronym, please use the long form and the acronym form together at the first use on a page. Example:
+**Guideline:** avoid acronyms and abbreviations where possible.
+
+Use them only where the acronym is easier to understand than the long form.
+
+Example: `SSH` is very common, `secure shell` is not widely understood and ambiguous.
+
+**Guideline:** When using an acronym/abbreviation, please use the long form first and then the acronym form in parantheses, to introduce the acronym.
+
+Example:
 
 > The Ingress Controller (IC) manages incoming traffic to your services.
 
-After that, you can use the acronym without the long form.
+After that, you can use the acronym without the long form and without parantheses.
 
 ### Headline title case
 
-We use [Title Case](https://titlecase.com/) for the main article headline, but not for lower level headlines.
+We use [Title Case](https://titlecase.com/) only for the main article headline, but not for lower level headlines.
 
 ### Code blocks and syntax highlighting
 
-For **code blocks**, we give language hints to ensure proper syntax highlighting. A YAML block, for example, is opened with triple back-ticks followed by `yaml`.
+**Guideline:** For code blocks, we give language hints to ensure proper syntax highlighting.
 
-However, shell commands and their output get the fake hint `nohighlight` to prevent any funky syntax highlighting.
+A YAML block, for example, is opened with triple back-ticks followed by `yaml`:
+
+    ```yaml
+    foo: bar
+    ```
+
+**Guideline:** shell commands and their output get the fake hint `nohighlight` to prevent any funky syntax highlighting.
```

Comment:
```suggestion
**Guideline:** Shell commands and their output get the fake hint `nohighlight` to prevent any funky syntax highlighting.
```

(consistency)